client/src/pages/ContestDetails/ContestDetailTabs/SubmissionsList.tsx

```diff
@@ -4,28 +4,36 @@ import ImageList from '@material-ui/core/ImageList';
 import ImageListItem from '@material-ui/core/ImageListItem';
 import ImageListItemBar from '@material-ui/core/ImageListItemBar';
 import useStyles from './useStyles';
+import { Submission } from '../../../interface/Submission';
 
-interface Props {
-  submissionList: Array<{ img: string; username: string }>;
+interface SubmissionListProps {
+  submissionList?: Submission[];
 }
-export function SubmissionsGrid({ submissionList }: Props): JSX.Element {
+
+export function SubmissionsGrid({ submissionList }: SubmissionListProps): JSX.Element {
   const classes = useStyles();
   return (
     <ImageList cols={4}>
-      {submissionList.map((submission: { img: string; username: string }) => (
-        <ImageListItem key={submission.img}>
-          <img
-            srcSet={`${submission.img}?w=248&fit=crop&auto=format 1x,
-                ${submission.img}?w=248&fit=crop&auto=format&dpr=2 2x`}
-            loading="lazy"
-          />
-          <ImageListItemBar title={'By @' + submission.username} className={classes.caption} />
-        </ImageListItem>
-      ))}
+      {submissionList?.map((submission: any) =>
+        submission.files.map((_file: string, _index: number) => (
```

Comment:
Please avoid usage of indexes as keys, can we use something more unique here?

Ref: https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9

Reply:
noted, changed to use file name